### PR TITLE
added deploy script

### LINKED ISSUE #30: Deploy Script (P1)
Add .js script which deploy all contracts and move abi and addresses to frontend catalog. 

Abi files and address are needed to map contract methods on frontend side. 

For better understanding check deploy Token file and see which .json files are being moved and handling basic token operations in frontend files.  Ethers docs could be useful https://docs.ethers.org/v5/api/contract/example/

I prepare some steps how it could look

1. addrTTPSC = new Token()
2. addrPay = new Payments(addrTTPSC)
3. Then you have to transfer funds (tokens) from the original token address to the manager address - addrPay
4. You create a contract addrRew = RewardsManager(addrPay)
5. Then move the built files to the appropriate path in the frontend subfolder

### PR DESCRIPTION
**Pull request by ad-jusk**

### CODE REVIEW

File: scripts/deployAll.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Function that deploys TTPSC token contract
+RETURN: TTPSC token
+*/
+const deployToken = async () => {
+
+    const Token = await ethers.getContractFactory("TTPSC");
+    const ttpscToken = await Token.deploy(
+        tokenArgs.TOKEN_NAME,
+        tokenArgs.TOKEN_SYMBOL,
+        tokenArgs.INITIAL_SUPPLY
```
Comment (MaciejIwan) [inline scripts/deployAll.js]: The initial value is passed in wei. Please multiply it by 10^18 (convert it to ether)
const weiValue = 100000;
const ethValue = ethers.utils.formatEther(weiValue);

File: scripts/deployAll.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+*/
+const deployPaymentsManager = async token => {
+    
+    const PaymentsManager = await ethers.getContractFactory("PaymentsManager");
+    const paymentsManager = await PaymentsManager.deploy(token.address);
+
+    await paymentsManager.deployed();
+    console.log("Payments manager contract deployed at address: ", paymentsManager.address);
+
+    await token.transfer(paymentsManager.address, tokenArgs.INITIAL_SUPPLY);
```
Comment (MaciejIwan) [inline scripts/deployAll.js]: Method extracting is not really neccessary at this moment, but add console.log() here

**Review by MaciejIwan — CHANGES_REQUESTED** [review]
I add some notes in your code.
Also add some informations to README file

File: scripts/deployAll.js (hunk 1 above)
Comment (ad-jusk) [reply scripts/deployAll.js]: Good point, fixed it.

File: scripts/deployAll.js (hunk 2 above)
Comment (ad-jusk) [reply scripts/deployAll.js]: Sure!

**Comment by ad-jusk** [comment]
@MaciejIwan fixes are done, please verify them

**Review by MaciejIwan — APPROVED** [review]

**Comment by MaciejIwan** [comment]
@ad-jusk good job! Let's merge

**Pull request merged**